usr/src/uts/common/fs/zfs/sys/zfs_debug.h

```diff
@@ -55,11 +55,10 @@ extern boolean_t zfs_free_leak_on_eio;
 #define	ZFS_DEBUG_DNODE_VERIFY		(1 << 2)
 #define	ZFS_DEBUG_SNAPNAMES		(1 << 3)
 #define	ZFS_DEBUG_MODIFY		(1 << 4)
-#define	ZFS_DEBUG_SPA			(1 << 5)
```

Comment:
I'd rather not remove this for the selfishly simple case that existing debug scripts outside of the OpenZFS tree have documented these and exposed them to people. Rather than removal, can we just add a /* deprecated */ comment?

Reply:
I think your concern is that you have scripts that are setting the 5th bit, and we could reuse this bit to mean something else in the future, so you'd like this to be renamed to `ZFS_DEBUG_DONT_USE` or something like that?

Reply:
precisely

Reply:
While I see this as a pretty deep internal detail that folks shouldn't be relying on, you could also argue that these should really be separate boolean_t tunables.  If we had done that, retiring one of them might break your script but it wouldn't do something surprising.  Given that there are plenty of bits, how about we remove `ZFS_DEBUG_SPA` but leave a comment saying `/* 1<<5 was previously used, try not to reuse */`.

Reply:
agree. It is more important that the existing, operable values remain <<6, <<7, ... than <<5 is used. One disadvantage of using bitmasks. This LGTM. Thanks